blackbox/docs/release_notes/1.1.4.txt

```diff
@@ -0,0 +1,78 @@
+.. _version_1.1.4:
+
+=============
+Version 1.1.4
+=============
+
+Released on 2017/06/02.
+
+.. NOTE::
+
+   If you are upgrading a cluster, you must be running CrateDB 0.57.0 or higher
+   before you upgrade to 1.1.4.
+
+   If you want to perform a :ref:`cluster_upgrade`, your current CrateDB
+   version number must be at least :ref:`version_1.1.1`. Any upgrade from a
+   version prior to this will require a full cluster restart.
```

Comment:
this last para should be:

```
   If you want to perform a :ref:`rolling_upgrade`, your current CrateDB
   version number must be :ref:`version_1.1.1` or higher. If you want to
   upgrade from a version prior to this, the upgrade will introduce all of the
   breaking changes listed for :ref:`version_1.1.0`, and will require a
   :ref:`full_restart_upgrade`.
```

I changed some of the text in these sections. I think maybe the release procedure needs to be updated to match the new text

I can do that if you like?